.buildkite/pipeline.yml

```
@@ -0,0 +1,9 @@
+steps:
+  - label: harmonic-aarch64-darwin
+    agents:
+      mac: 1
+      system: aarch64-darwin
```

Probably need to also specify you want nix:

```suggestion
      system: aarch64-darwin
      nix: 1
```